src/main/java/com/videoclub/suriken/service/impl/MovieServiceImpl.java

```diff
@@ -0,0 +1,79 @@
+package com.videoclub.suriken.service.impl;
+
+import com.videoclub.suriken.model.Movie;
+import com.videoclub.suriken.model.MovieRenter;
+import com.videoclub.suriken.repository.MovieRepository;
+import com.videoclub.suriken.repository.RenterRepository;
+import com.videoclub.suriken.service.MovieService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+
+    private final MovieRepository movieRepository;
+    private final RenterRepository renterRepository;
+
+    @Autowired
+    public MovieServiceImpl(MovieRepository movieRepository, RenterRepository renterRepository) {
+        this.movieRepository = movieRepository;
+        this.renterRepository = renterRepository;
+    }
+
+
+    @Override
+    public void addMovieToDb(Movie movie) {
+        movie.setMovGenre(Movie.Genre.Drama);
```

Comment:
Zasto je svaki film drama?

Reply:
Zbog onog problem prosledjivanja iz json-a. Svi zanrovi su mi bili prazni, pa sam dodao da budu drama da mi ne bi bilo null u tabelama. To mi je da kazem jedini problem na koji sam naisao a da ga nisam resio!